website/docs/pipelines/azure-login.md

```diff
@@ -0,0 +1,106 @@
+---
+sidebar_position: 1
+sidebar_label: Azure Login
+---
+
+# Azure Login
+
+There are several ways to log into Azure by a GitHub Action. The our chosen method doesn't require maintenance or secret management as it is a passwordless approach. Once set up, it works.
+
+Only two components are needed to log into Azure:
+
+- An Azure Managed Identity resource
+- A federation between the Azure Managed Identity and GitHub
+
+The folder `infra/identity` contains the Terraform definition, often created by Cloud Engineers using a [Terraform custom module](https://github.com/pagopa/dx/tree/main/infra/modules/azure_federated_identity_with_github) to create both the Managed Identity and the federation with GitHub. Module also provides a set of default roles in the current subscription, which are likely to be enough for new repositories - more on this later.
```

Comment:
why 'often'? in which cases is not created by cloud eng?

Reply:
how are you supposed to engage cloud eng andh when?

Reply:
@gunzip implementing `infra/identity` is trivial and no support from `cloud engineers` is needed, cloud eng should maintain the linked module

Reply:
I'd remove 'often created by Cloud Engineers' in this case

Reply:
The ideal path would include the Terraform identity definition written by developers, and the PR review and the Terraform apply done by cloud engs (devs can create identities but not giving them roles)